protos/flyteidl/core/tasks.proto

```diff
@@ -263,3 +264,11 @@ message K8sObjectMetadata {
   // Optional annotations to add to the pod definition.
   map<string, string> annotations = 2;
 }
+
+// Sql represents a generic sql workload with a statement and engine.
+message Sql {
+    // Define a query to execute.
+    string statement = 1;
+    // Define the engine to execute SQL statement. For example hive or bigquery.
+    string engine = 2;
```

Comment:
What does engine mean, please explain

Reply:
@EngHabu @kanterov could you help explain this field?  How will propeller use this engine field vs the task type?  Or is it for a specific plugin to use because a given task type might have multiple engines?